src/python/detectors/notebook_best_practice_violation/notebook_best_practice_violation_compliant.py

```diff
@@ -0,0 +1,18 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This rule detects three kinds of violations: empty cell, too many cel, as well as misplaced imports. You need to add them.

Reply:
Improved examples.